### PR TITLE
[15.0][MIG] product_abc_classification: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT38328

@chienandalu @pedrobaeza please review!

### CODE REVIEW

File: product_abc_classification/README.rst [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+* ForgeFlow
+
+Contributors
+~~~~~~~~~~~~
+
+* Miquel Raïch <miquel.raich@forgeflow.com>
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Carlos Roca
+  * Stefan Ungureanu
```
Comment (pedrobaeza) [inline product_abc_classification/README.rst:83]: You shouldn't edit this manually, but run the OCA tool to generate the README. If not able to run, simply don't touch and let the bot to modify it on merge.

**Review by pedrobaeza — APPROVED** [review]
Ouch, a page in a group in previous versions...

**Review by chienandalu — APPROVED** [review]
Aside from comments :+1: Thanks!

File: product_abc_classification/README.rst (hunk 1 above)
Comment (stefan-tecnativa) [reply product_abc_classification/README.rst:83]: I am aware that the README should not be modified in this way, I must have lost my way.

**Comment by chienandalu** [comment]
This is ready for merge I think

**Pull request merged**